code/modules/mining/equipment/wormhole_jaunter.dm

```diff
@@ -89,3 +89,91 @@
 			if(ishuman(L))
 				shake_camera(L, 20, 1)
 				addtimer(CALLBACK(L, /mob/living/carbon.proc/vomit), 20)
+
+/obj/item/wormhole_jaunter/contractor
+	name = "emergency extraction flare"
+	icon = 'icons/obj/lighting.dmi'
+	desc = "A single use extraction flare, that will let you create a portal to any beacon on the station. You must chose the destination before hand, else it will head to a random beacon. Portal is made 5 seconds after activation, and has 1 use."
```

Comment:
```suggestion
	desc = "A single-use extraction flare that will let you create a portal to any beacon on the station. You must choose the destination beforehand, else it will target a random beacon. The portal is generated 5 seconds after activation, and has 1 use."
```